### PR TITLE
server.c: fix leak while freeing rdpPeerChannel->queue

### PR DESCRIPTION
**Pull request by kubistika**
in server.c:92 a message is malloced and posted to the queue, but if won't be processed, a leak will occur when MessageQueue_Free will be called.

### CODE REVIEW

File: libfreerdp/core/server.c [hunk 1]
```diff
@@ -1033,6 +1038,7 @@ HANDLE WINAPI FreeRDP_WTSVirtualChannelOpen(HANDLE hServer, DWORD SessionId, LPS
 
 	if (!channel)
 	{
+		wObject queueCallbacks;
```
Comment (akallabeth) [inline libfreerdp/core/server.c]: Missing initialization `wObject queueCallbacks = { 0 };`

Comment (kubistika) [reply libfreerdp/core/server.c]: fixed it, thank you :))

**Comment by kubistika** [comment]
@akallabeth nice!

**Comment by kubistika** [comment]
@akallabeth should we merge this? ^^

**Pull request merged**